### PR TITLE
[wasm][tests] Fix, and simplify sample building on CI

### PR DESCRIPTION
**Pull request by radical**
1. Instead of hardcoding specific command lines for wasm samples in `sendtohelixhelp.proj`, generate a run script, which allows moving the execution commands to the respective projects.
2. this enables running the library tests additionally for v8, and browser, in case of `runtime-manual`. This was recently changed to run only for nodejs
3. And avoids running samples on debugger tests build

### CODE REVIEW

File: eng/pipelines/runtime-manual.yml [hunk 1]
```diff
@@ -488,16 +488,16 @@ jobs:
...
-          eq(dependencies.evaluate_paths.outputs['SetPathVars_installer.containsChange'], true),
+          eq(variables['isManualOrIsNotPR'], true),
           eq(variables['isFullMatrix'], true))
       # extra steps, run tests
       extraStepsTemplate: /eng/pipelines/libraries/helix.yml
       extraStepsParameters:
         creator: dotnet-bot
         testRunNamePrefixSuffix: Mono_$(_BuildConfig)
         scenarios:
+        - normal
```
Comment (pavelsavara) [inline eng/pipelines/runtime-manual.yml:499]: Comment on the section said NodeJs, should we fix it ?

File: eng/pipelines/runtime-manual.yml [hunk 2]
```diff
@@ -488,16 +488,16 @@ jobs:
       timeoutInMinutes: 180
       condition: >-
         or(
-          eq(dependencies.evaluate_paths.outputs['SetPathVars_libraries.containsChange'], true),
-          eq(dependencies.evaluate_paths.outputs['SetPathVars_mono.containsChange'], true),
-          eq(dependencies.evaluate_paths.outputs['SetPathVars_installer.containsChange'], true),
+          eq(variables['isManualOrIsNotPR'], true),
```
Comment (pavelsavara) [inline eng/pipelines/runtime-manual.yml:491]: The same  conditions you removed are duplicated in many other places of this file. Should we remove them too ?

**Review by pavelsavara — APPROVED** [review]
LGTM, my comments are more questions than feedback.

File: eng/pipelines/runtime-manual.yml (hunk 1 above)
Comment (radical) [reply eng/pipelines/runtime-manual.yml:499]: I have a follow up PR, I'll remove the comment in that.

File: eng/pipelines/runtime-manual.yml (hunk 2 above)
Comment (radical) [reply eng/pipelines/runtime-manual.yml:491]: @steveisok would you prefer that? Or should I restore the conditions, and just add the `manual` one?

Comment (steveisok) [reply eng/pipelines/runtime-manual.yml:491]: I think that's ok.

**Comment by radical** [comment]
Failures are unrelated.

**Review by steveisok — APPROVED** [review]

**Pull request merged**